vars/packer.groovy

```diff
@@ -158,7 +167,11 @@ def call(body) {
       config.get('cookbookS3Bucket'),
       config.get('cookbookS3Region', region),
       config.get('cookbookS3Path'))
-    ptb.addChefSoloProvisioner(config.runList,config.get('chefJSON'),config.get('chefVersion'))
+    if (config.get('useCinc', false)) {
```

Comment:
no need for the if statement, you could just pass the value of `config.get('useCinc', false)` in to the method as the method handles the logic

```groovy
ptb.addChefSoloProvisioner(config.runList,config.get('chefJSON'),config.get('cincVersion'), config.get('useCinc', false))